mobile/lib/pages/common/splash_screen.page.dart

```diff
@@ -10,65 +9,80 @@ import 'package:immich_mobile/entities/store.entity.dart';
 import 'package:logging/logging.dart';
 
 @RoutePage()
-class SplashScreenPage extends HookConsumerWidget {
+class SplashScreenPage extends StatefulHookConsumerWidget {
   const SplashScreenPage({super.key});
 
   @override
-  Widget build(BuildContext context, WidgetRef ref) {
+  SplashScreenPageState createState() => SplashScreenPageState();
+}
+
+class SplashScreenPageState extends ConsumerState<SplashScreenPage> {
+  final log = Logger("SplashScreenPage");
+  @override
+  void initState() {
```

Comment:
Is there a particular reason this is stateful? It doesn't seem like there's actually any state.

Reply:
I find it cleaner to use stateful here to reason the logic of the initialization of the widget/page in this case